Makefile.core.mk

```diff
@@ -274,6 +274,7 @@ BINARIES:=./istioctl/cmd/istioctl \
   ./operator/cmd/operator \
   ./cni/cmd/istio-cni \
   ./cni/cmd/istio-cni-repair \
+  ./cni/cmd/istio-cni-taint \
```

Comment:
why does it run as its own binary? instead of as part of istio-cni or istio-cni-repair?

Reply:
At the moment this is still experimental functionality, so I think keeping it as a separate binary in the same daemonset is safer since we can use a flag to add/remove the entire container in the manifests.

Reply:
Adding a new binary brings a (small, but real) cost to users  and developers. We can put feature flags in the code

Reply:
Notice how cni has 5 binaries and Istiod has 1... Not a big deal though

Reply:
This adds 41mb to the docker file which is downloaded on every single node